build.sbt

```diff
@@ -817,6 +815,7 @@ lazy val test = project
     fork in IntegrationTest := true,
     // enable this in 2.13, when tests pass
     //scalacOptions in Compile += "-Yvalidate-pos:parser,typer",
+    scalacOptions -= "-Ywarn-unused:imports",
```

Comment:
I'm fine with doing this, but how come you did it?

Reply:
I don't know how reason, but `partest` fails when adding `-Ywarn-unused: imports`.

Reply:
The 2.13 PR does something similar. Test compilations receive these args, too, with fatal warnings.